src/common/utils/DelayedJobs.ts

```diff
@@ -0,0 +1,70 @@
+export class DelayedJobs {
```

Comment:
All the 'schedule/cancel delayed' logic is now moved into a separate entity called DelayedJobs. This way we unload some burden from within the ProjectService impl.